web/src/message_list_view.js

```diff
@@ -1003,6 +1003,13 @@ export class MessageListView {
             }
         }
 
+        const is_conversation_view = this.list.data.filter.is_conversation_view();
+        if (is_conversation_view) {
+            $("#zfilt").addClass("is-conversation-view");
+        } else {
+            $("#zfilt").removeClass("is-conversation-view");
+        }
```

Comment:
This is wrong if we are rerendering `message_lists.home`, which should not mutate `zfilt`. Please look at the `table_name` attribute for what part of the DOM to interact with.

Reply:
Ill make the adjustments and update the code. Thank you for your feedback on this :smile: